packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/choice/accessories-choice.html

```diff
@@ -0,0 +1,166 @@
+                        <oui-dropdown-item
+                            data-on-click="$ctrl.sortPriceDesc()"
+                        >
+                            <span
+                                data-translate="telephony_line_phone_accessories_filter_price_desc"
+                            ></span>
+                        </oui-dropdown-item>
+                    </oui-dropdown-content>
+                </oui-dropdown>
+                <oui-dropdown data-ng-if="$ctrl.phone" align="end">
+                    <oui-dropdown-trigger
+                        text="{{ ::'telephony_line_phone_accessories_filter_compatibility' | translate }}"
+                    >
+                    </oui-dropdown-trigger>
+                    <oui-dropdown-content>
+                        <oui-dropdown-item
+                            data-on-click="$ctrl.filterWithPhone()"
+                        >
+                            <span
+                                data-translate="telephony_line_phone_accessories_filter_compatibility_compatible"
+                            ></span>
+                        </oui-dropdown-item>
+                        <oui-dropdown-item data-on-click="$ctrl.filterAll()">
+                            <span
+                                data-translate="telephony_line_phone_accessories_filter_compatibility_all"
+                            ></span>
+                        </oui-dropdown-item>
+                    </oui-dropdown-content>
+                </oui-dropdown>
+            </div>
+
+            <!-- Define tabs -->
+            <oui-tabs>
+                <oui-tabs-item
+                    data-ng-repeat="brand in $ctrl.brandList track by $index"
+                    data-heading="{{ ::brand }}"
+                    data-on-active="$ctrl.filterByBrand(brand)"
+                >
+                    <!-- Define the thumbnail for accessories list -->
+                    <div class="row">
+                        <div
+                            class="col-xs-12 col-sm-6 col-md-3 accessory-item"
+                            data-ng-repeat="accessory in $ctrl.accessoriesDisplayed track by accessory.name"
+                        >
+                            <oui-radio-group
+                                data-model="$ctrl.accessoryToOrder"
+                                data-on-change="$ctrl.selectAccessory()"
+                            >
+                                <oui-radio
+                                    thumbnail
+                                    variant="light"
+                                    data-value="accessory.name"
+                                >
+                                    <div class="thumbnail">
+                                        <img
+                                            class="sized"
+                                            data-ng-if="accessory.img"
+                                            alt="{{accessory.description}}"
```

Comment:
I think you need to 
```suggestion
                                            ng-attr-alt="{{accessory.description}}"
```
in that case

Reply:
I would like but when I try to commit this, I have this error :
An alt attribute must be present on <img> elements. (alt-require)
So I can't do what you asked.

Reply:
Therefore there should be two attributes, an empty `alt` and a `ng-attr-alt` with the value